pubsub/subscriptions/create_message_transforms.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package subscriptions
+
+// [START pubsub_create_subscription_with_smt]
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"cloud.google.com/go/pubsub/v2"
+	"cloud.google.com/go/pubsub/v2/apiv1/pubsubpb"
+)
+
+// createSubscriptionWithSMT creates a subscription with a single message transform function applied.
+func createSubscriptionWithSMT(w io.Writer, projectID, topicID, subID string) error {
```

Comment:
The example function signature is different than the [V1 version](https://github.com/GoogleCloudPlatform/golang-samples/blob/d3bdadadac8b9ff046d23249d3ce19681f8e1061/pubsub/v1samples/subscriptions/create_message_transforms.go#L27). Is that by design?